setup.py

```diff
@@ -49,7 +49,9 @@
         'jinja2 >= 2.7.2',
     ],
     extras_require={
-        'yaml': [pyyaml_version,]
+        'yaml': [pyyaml_version,],
+        'ansible': ['ansible >= 2.9.10',],
+        'all': [pyyaml_version, 'ansible >= 2.9.10',],
```

Comment:
may be, if we can provide a list of what are the filters that we want to install, and if its empty we can just install yaml

Reply:
I don't know how to do this. It's also oht of skope for this PR.

If you have a prototype idea in mind, feel free to open a PR against master iny fork to discuss and elaborate.